download/upgradeRecipe/upgradeRecipe6.1.adoc

```diff
@@ -1030,14 +1030,14 @@ and no longer alters the filename.
 [.upgrade-recipe-minor]
 === `<subChainChangeMoveSelector>` and `<subChainSwapMoveSelector>`: default parameters changed
 
-`<subChainChangeMoveSelector>` and `<subChainSwapMoveSelector>` 's `<subChainSelector>`s now default
+`<subChainChangeMoveSelector>` and ``<subChainSwapMoveSelector>``'s ``<subChainSelector>``s now default
```

Comment:
Thanks for the PR :)

Interesting trick to fix the 's problem. I wonder if it can also be solved by adding a backslash in front of the '. This is one to watch when we're migrating the optaplanner-docs from docbook to asciidoc later this month: I believe the ref manual does that after <literals> regularly...